boards/apollo3/lora_things_plus/src/main.rs

```diff
@@ -0,0 +1,490 @@
+        components::spi::SpiMuxComponent::new(&peripherals.iom3, dynamic_deferred_caller).finalize(
+            components::spi_mux_component_helper!(apollo3::iom::Iom<'static>),
+        );
+
+    // Setup BLE
+    mcu_ctrl.enable_ble();
+    clkgen.enable_ble();
+    pwr_ctrl.enable_ble();
+    let _ = &peripherals.ble.setup_clocks();
+    mcu_ctrl.reset_ble();
+    let _ = &peripherals.ble.power_up();
+    let _ = &peripherals.ble.ble_initialise();
+
+    let ble_radio = ble::BLEComponent::new(
+        board_kernel,
+        capsules::ble_advertising_driver::DRIVER_NUM,
+        &peripherals.ble,
+        mux_alarm,
+    )
+    .finalize(());
+
+    mcu_ctrl.print_chip_revision();
+
+    debug!("Initialization complete. Entering main loop");
+
+    // These symbols are defined in the linker script.
+    extern "C" {
+        /// Beginning of the ROM region containing app images.
+        static _sapps: u8;
+        /// End of the ROM region containing app images.
+        static _eapps: u8;
+        /// Beginning of the RAM region for app memory.
+        static mut _sappmem: u8;
+        /// End of the RAM region for app memory.
+        static _eappmem: u8;
+    }
+
+    let scheduler = components::sched::round_robin::RoundRobinComponent::new(&PROCESSES)
+        .finalize(components::rr_component_helper!(NUM_PROCS));
+
+    let systick = cortexm4::systick::SysTick::new_with_calibration(48_000_000);
+
+    let artemis_nano = static_init!(
+        LoRaThingsPlus,
+        LoRaThingsPlus {
+            alarm,
+            console,
+            gpio,
+            led,
+            i2c_master,
+            external_spi_controller,
+            ble_radio,
+            temperature,
+            humidity,
+            air_quality,
+            scheduler,
+            systick,
+        }
+    );
+
+    let chip = static_init!(
+        apollo3::chip::Apollo3<Apollo3DefaultPeripherals>,
+        apollo3::chip::Apollo3::new(peripherals)
+    );
+    CHIP = Some(chip);
+
+    kernel::process::load_processes(
+        board_kernel,
+        chip,
+        core::slice::from_raw_parts(
+            &_sapps as *const u8,
+            &_eapps as *const u8 as usize - &_sapps as *const u8 as usize,
+        ),
+        core::slice::from_raw_parts_mut(
+            &mut _sappmem as *mut u8,
+            &_eappmem as *const u8 as usize - &_sappmem as *const u8 as usize,
+        ),
+        &mut PROCESSES,
+        &FAULT_RESPONSE,
+        &process_mgmt_cap,
+    )
+    .unwrap_or_else(|err| {
+        debug!("Error loading processes!");
+        debug!("{:?}", err);
+    });
+
+    (board_kernel, artemis_nano, chip, peripherals)
+}
+
+/// Main function.
+///
+/// This function is called from the arch crate after some very basic RISC-V
+/// setup and RAM initialization.
+#[no_mangle]
+pub unsafe fn main() {
+    #[cfg(test)]
+    test_main();
+
+    #[cfg(not(test))]
+    {
+        let (board_kernel, esp32_c3_board, chip, _peripherals) = setup();
```

Comment:
```suggestion
        let (board_kernel, sf_lora_thing_plus_board, chip, _peripherals) = setup();
```
Should probably change this here too https://github.com/tock/tock/blob/09600e9deab75e1b3faa7eec2be26cbebae2d2cf/boards/redboard_artemis_nano/src/main.rs#L433